### ISSUE TITLE
Move list of hungarian prefixes to somewhere user configurable

### ISSUE THREAD

**Issue by philoushka** [philoushka/dehungarian#3]
Currently the list of prefixes is hardcoded at [line 19](https://github.com/philoushka/dehungarian/blob/master/dehungarian/dehungarian/DehungarianAnalyzer.cs#L19) .

This should be moved somewhere accessible for the user. Perhaps they want to modify this list to suit their needs.

**Comment by PawelTroka** [comment]
Yes, this is very important. For example I have often encountered "co" prefix, probably for "class object".
